websites/A/Anime Planet/presence.ts

```diff
@@ -244,7 +237,7 @@ presence.on("UpdateData", async () => {
         },
         {
           label: (await strings).viewSeries,
-          url: document.querySelector<EHref>("h2.sub > a").href
+          url: document.querySelector<ElementHref>('h2.sub > a').href
```

Comment:
Already told you to change it.
![image](https://user-images.githubusercontent.com/9348108/111979579-8953cd00-8b05-11eb-8e08-b517e2b31ab8.png)

Reply:
![image](https://user-images.githubusercontent.com/54318514/111979688-f41dfc00-8b37-11eb-864c-af90af06aec7.png)

Reply:
ekhem, ``?.href``

Reply:
or use ``HTMLHyperlinkElementUtils`` type.

Reply:
Still the same

Reply:
It's not like I want to use custom type also lol

Reply:
![image](https://user-images.githubusercontent.com/54318514/111980804-617e5c80-8b39-11eb-8645-d0471443d27a.png)

Reply:
I hates it when Github comments isn't real-time reee

Reply:
Ok it works with `HTMLAnchorElement`

Reply:
gud 👍